core/arch/arm/pta/sta_benchmark.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2015, Linaro Limited
```

Comment:
minor: 2017

could you rename **sta_benchmark.c** as **pta_benchmarck.c**.  "static TA" is a deprecated reference.